LanguageApp/app/screens/ScenariosScreen.js

```diff
@@ -33,7 +33,13 @@ function ScenariosScreen(props) {
           <ListItem
             title={item.title}
             prompt={item.prompt}
-            onPress={() => console.log("pressed", item)}
+            onPress={() => {
+              if (route.params.user_type === "CP") {
+                navigation.navigate(routes.PROVIDER_SCENARIO, item);
+              } else {
+                navigation.navigate(routes.LEARNER_SCENARIO, item);
```

Comment:
I believe `item` needs to be passed as `{ item }` in order to pass the object, otherwise, it is undefined. 

I ran a test in the ProviderScenarioScreen with a simple  `console.log(route.params.item);` and passing the object was successful

Reply:
can access item object using `route.params` instead of `route.params.item`